kube_hunter/conf/parser.py

```diff
@@ -54,4 +59,7 @@ def parse_args():
 
     parser.add_argument("--network-timeout", type=float, default=5.0, help="network operations timeout")
 
-    return parser.parse_args()
+    args = parser.parse_args()
+    if args.cidr:
+        args.cidr = args.cidr.replace(" ", "").split(",")
```

Comment:
Is this `if` statement really needed?
Removing it will make the `cidr` property an empty list (instead of an empty string).
```suggestion
    args.cidr = args.cidr.replace(" ", "").split(",")

```

Reply:
It's acually a NoneType since we didn't include default value to it.
so by changing this line it will end up with an AttributeError.
We can add a default value to the config.cidr, but it will just make our config object bigger for no reason